uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -212,6 +238,10 @@ sub createTarchiveArray {
                       "This seems not to be a valid archive for this study!".
                       "\n\n";
         $this->writeErrorLog($message, 3);
+	# no $tarchive nor $tarchive_id can be fetched since the archive 
+	# could not be found, making $upload_id hard to trace, so send some 
+	# random number (99999) instead for $upload_id in the notification_spool
+        $this->spool($message, 'Y', 99999);
```

Comment:
this is the only place where we would not be able to fetch the upload_id from the tarchive_id as the tarchive can not be found, so I chose to spool this number... please verify this is acceptable